projects/storefrontapp-e2e-cypress/cypress/integration/mobile/product-configuration.e2e-spec.ts

```diff
@@ -0,0 +1,33 @@
+import * as configuration from '../../helpers/product-configuration';
+import { formats } from '../../sample-data/viewports';
+
+const electronicsShop = 'electronics-spa';
+const testProduct = 'CONF_CAMERA_SL';
+
+// UI types
+const radioGroup = 'radioGroup';
+
+const CAMERA_DISPLAY = 'CAMERA_DISPLAY';
+const CAMERA_MODE = 'CAMERA_MODE';
+
+context('Product Configuration', () => {
+  beforeEach(() => {
+    cy.visit('/');
+  });
+
+  describe('Group Handling', () => {
+    it('should navigate using the group menu in mobile resolution', () => {
+      cy.window().then((win) => win.sessionStorage.clear());
+      cy.viewport(formats.mobile.width, formats.mobile.height);
```

Comment:
We no longer need separate files and folders for mobile and desktop e2e, since `cy.viewportContext()` helper was introduced in our repo. It allows for easy running the same e2e tests in desktop and mobile. Moreover some parts of the test that are specific only to mobile, can be wrapped in a helper `cy.onMobile(() => { ...`

See example:
https://github.com/SAP/spartacus/blob/38ede2849cfe1dfb1872840465ff5277d3fa40cf/projects/storefrontapp-e2e-cypress/cypress/integration/regression/added-to-cart-modal.e2e-spec.ts#L7-L8

https://github.com/SAP/spartacus/blob/38ede2849cfe1dfb1872840465ff5277d3fa40cf/projects/storefrontapp-e2e-cypress/cypress/integration/regression/added-to-cart-modal.e2e-spec.ts#L83-L86

Reply:
The for debugging purposes you might want to temporarily specify the only one viewport that should be used to run the tests. Then there is no need to modify the code of the tests. It suffices to set the Cypress env variable `'VIEWPORT'` in the appropriate `cypress.<...>.json` file. For example:

```json
{
  /* ... */
  "env": {
    "VIEWPORT": "mobile"
  }
}
```

Reply:
Can be postponed. Please create a ticket for it.

Reply:
Yes, we do have an issue for that: #11199